src/Maker/MakeCrud.php

```diff
@@ -247,5 +268,15 @@ public function configureDependencies(DependencyBuilder $dependencies)
             ParamConverter::class,
             'annotations'
         );
+
+        $dependencies->addClassDependency(
+            CssSelectorConverter::class,
+            'css-selector'
+        );
+
+        $dependencies->addClassDependency(
+            Client::class,
+            'browser-kit'
+        );
```

Comment:
These should not be required dependencies - the test are really an optional thing. Instead, IF these dependencies are installed, let's generate a test. If they are not installed, let's *not* generate the tests files, but print a message like:

> Skipping test generation because the test dependencies (css-select and browser-kit) are not installed.

Reply:
I've made this optional and added the notice.